.github/workflows/main.yml

```diff
@@ -0,0 +1,30 @@
+name: main
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 14
+      - run: npm run build
+      - id: package-version
+        uses: martinbeentjes/npm-get-version-action@v1.1.0
+      - uses: josephbmanley/butler-publish-itchio-action@v1.0.2
+        env:
+          BUTLER_CREDENTIALS: ${{ secrets.BUTLER_CREDENTIALS }}
+          CHANNEL: web
+          ITCH_GAME: borksy
+          ITCH_USER: ayolland
+          PACKAGE: docs
+          VERSION: ${{ steps.package-version.outputs.current-version}}
+      - uses: EndBug/add-and-commit@v7.3.0
+        with:
+          add: 'docs'
+          message: 'update build [skip ci]'
```

Comment:
this bit is to allow github pages to be published off the main branch. this will need to be configured under https://github.com/Ayolland/borksy/settings/pages by setting the "source" to `branch: main` `/docs`

(note that i haven't tested this bit; may need some additional changes if it doesn't work as expected on the first push)